### PR TITLE
Applying new media query mixin

### PR DESCRIPTION
**Pull request by chris-pearce**

### CODE REVIEW

File: layout/_l-grid-pull.scss [hunk 1]
```diff
@@ -154,7 +174,7 @@
 	 */
 
 	@if $grid-pull-classes-at-breakpoint-lap {
```
Comment (AlecRaeside) [inline layout/_l-grid-pull.scss:176]: cool, i'll move these into a loop after this is merged

File: utilities/_u-widths.scss [hunk 2]
```diff
@@ -12,46 +36,59 @@
...
    */
+
   .u-#{$namespace}one-half {width: $one-half;}
 
+
   /**
    * Thirds.
    */
+
   .u-#{$namespace}one-third {width: $one-third;}
```
Comment (AlecRaeside) [inline utilities/_u-widths.scss:54]: might be someway to put all of this in a loop in a mixin but I'm not sure, will need to research.

File: layout/_l-grid.scss [hunk 3]
```diff
@@ -176,7 +185,7 @@ $friendly-box-model: false !default;
 }
 
 // Only for lap plus (non-palm)
-@include respond-min(641) {
+@include respond-to(lap) {
  .grid--table-for-lap-plus {
```
Comment (AlecRaeside) [inline layout/_l-grid.scss]: `lap-plus` feels inconsistent, we're not referring to it elsewhere.

File: layout/_l-grid-pull.scss (hunk 1 above)
Comment (chris-pearce) [inline layout/_l-grid-pull.scss:176]: Yeah that would be ace.

File: utilities/_u-widths.scss (hunk 2 above)
Comment (chris-pearce) [inline utilities/_u-widths.scss:54]: Cool.

**Comment by AlecRaeside** [comment]
LGTM

**Pull request merged**